arenadata/docker-compose.yaml

```diff
@@ -4,19 +4,20 @@ version: "3"
 services:
   #image can be either hub.adsw.io/library/gpdb6_regress:${BRANCH_NAME}-x86-64 or
   #hub.adsw.io/library/gpdb6_regress:${BRANCH_NAME}-ppc64le depending on the arch
-  mdw:
+  cdw:
     image: "${IMAGE}"
     working_dir: /home/gpadmin
-    hostname: mdw
+    hostname: cdw
     volumes:
       - "$PWD/ssh_keys/id_rsa:/home/gpadmin/.ssh/id_rsa"
       - "$PWD/ssh_keys/id_rsa.pub:/home/gpadmin/.ssh/id_rsa.pub"
       - "$PWD/allure-results:/tmp/allure-results"
     privileged: true
     sysctls:
       kernel.sem: 500 1024000 200 4096
+    init: true
     entrypoint: >
-      bash -c "set -e && tail -f /home/gpadmin/gpdb_src/README.md"
```

Comment:
Is this line equivalent to `sleep infinity`? Does enabling init increase overhead?

Reply:
yes, it is equivalent, but in rocky 8 it does not work the way it does in centos 7. the task of the command `bash -c "set -e && tail -f /home/gpadmin/gpdb_src/README.md"` is to start an infinite wait and that the process with pid 1 is bash (for the correct completion of "orphaned" processes) but in rocky 8, pid 1 gets tail (possibly due to some new bash optimizations) and because of that, zombie processes begin to appear.
I don't think the init process will add an overhead.